test/users/rbarrett/diffuse2d-v3-blc.chpl

```diff
@@ -410,29 +410,29 @@ proc ApplyStencil3d27pt ( ProblemSpace2d, newGrid, oldGrid, iteration ) {
                      oldGrid(ijk+frontsouthwest) + oldGrid(ijk+frontsouth) + oldGrid(ijk+frontsoutheast) )
 
                   / 27.0;
-                  
+
    // Check answer.
-   
+
    CheckAnswer ( ProblemSpace2d, newGrid, iteration );
-   
+
 } // End function ApplyStencil3d27pt.
 
 /* ====================================================================================================== */
 
 proc CheckAnswer ( ProblemSpace2d, newGrid, iteration ) {
-   
+
    /* ------------------ */
    /* Local Declarations */
    /* ------------------ */
-   
+
    var
       err : real;  // Computed error.
 
    /* --------------------- */
    /* Executable Statements */
    /* --------------------- */
 
-   if ( verbose ) 
+   if ( verbose )
```

Comment:
Here is another example where there are 4 significant changes and 80+ (?) whitespace adjustments.